### PR TITLE
Add translation to untranslated strings in templates

### LINKED ISSUE #9678: The word "required" is not translatable
- [ X ] I have searched open and closed issues for duplicates

### Bug Description

According to [these two lines in the source code](https://github.com/friendica/friendica/blob/2020.12-rc/src/Module/Admin/Blocklist/Server.php#L79-L80) the word "reqiured" is appended to the translated string as is.

### Steps to Reproduce

1.  Choose an other language than English.
2.  Go to Admin -> Server Blocklist and see the form.

Actual Result:

Translated: Kiszolgálótartomány-minta required, Original English: Server Domain Pattern required
Translated: A tiltás oka required, Original English: Reason for the block reqiured

Expected Result:

All strings should be translated. It would be nice to separate "required" a little bit, for example:

- Server domain pattern (required)
- Reason for the block (reqiured)

Please note, that one string is Title Case while the other is Sentence case. This is not consistent.

### LINKED ISSUE #9682: "Yourself" is not marked as translatable in the profile page
- [ X ] I have searched open and closed issues for duplicates

### Bug Description

In the profile page the option "Yourself" is not marked as translatable. You can [see the string here](https://github.com/friendica/friendica/blob/2020.12-rc/view/templates/profile/index.tpl#L105).

### Steps to Reproduce

1. Change language other than English (let assume, the seleced language is fully translated)
2. Go to the profile and scroll down to the bottom. See the first option of "View profile as" field, it is always in English

Actual Result:

The string is displayed always in English.

Expected Result:

The string should be displayed in the selected language of the user interface.

### PR DESCRIPTION
**Pull request by MrPetovan**
Fixes #9678 
Fixes #9682

Thanks to @urbalazs for looking for the untranslated strings in the code, it helped.

### CODE REVIEW

**Comment by annando** [comment]
Concerning "required": I think that it is meant to serve as an HTML attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required

So yes, it should help marking the input field as "required", but also the attribute should be set.

**Comment by MrPetovan** [comment]
Neither the original issue nor this PR are about the `required` attribute that was already set and still will.

File: view/theme/frio/templates/field_input.tpl [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
 	<div id="id_{{$field.0}}_wrapper" class="form-group field input">
 	{{if !isset($label) || $label != false }}
-		<label for="id_{{$field.0}}" id="label_{{$field.0}}">{{$field.1 nofilter}}{{if $field.4}}<span class="required"> {{$field.4}}</span>{{/if}}</label>
+		<label for="id_{{$field.0}}" id="label_{{$field.0}}">{{$field.1 nofilter}}{{if $field.4}} <span class="required" title="{{$field.4}}">*</span>{{/if}}</label>
 	{{/if}}
-		<input class="form-control" name="{{$field.0}}" id="id_{{$field.0}}"{{if $field.6}} type="{{$field.6}}"{{else}} type="text"{{/if}} value="{{$field.2}}"{{if $field.4 eq "required"}} required{{/if}}{{if $field.5 eq "autofocus"}} autofocus{{elseif $field.5}} {{$field.5 nofilter}}{{/if}} aria-describedby="{{$field.0}}_tip">
+		<input class="form-control" name="{{$field.0}}" id="id_{{$field.0}}" type="{{$field.6|default:'text'}}" value="{{$field.2}}"{{if $field.4}} required{{/if}}{{if $field.5 eq "autofocus"}} autofocus{{elseif $field.5}} {{$field.5 nofilter}}{{/if}} aria-describedby="{{$field.0}}_tip">
```
Comment (annando) [inline view/theme/frio/templates/field_input.tpl:6]: In the old code there is this check for `{{if $field.4 eq "required"}}`. Now we seem to assume, that `$field.4` always is the "required" field?

Comment (MrPetovan) [reply view/theme/frio/templates/field_input.tpl:6]: Because it already was, see the current field documentation. Not all input fields use it, though.

**Pull request merged**